server/api/tags_apis.py

```diff
@@ -26,6 +28,58 @@ def get(self):
             current_app.logger.critical(error_msg)
             return {"error": error_msg}, 500
 
+    @token_auth.login_required
+    def post(self):
```

Comment:
You should use the delete verb rather than post here.  You could also pass the tag through as a param, which will save you fishing it out of the request, eg

`def delete(self, org_tag: str)`

Reply:
i initially tried using delete. but then i think if the organisation tag is quite long, it might cause trouble. however If what we pass is an ID then should not be a problem. but from the problem description, it seems like the thing to be passed is the organisation tag. what do you think?

Reply:
@fiennyangeln I saw your comment re the fact that insert code handles identical dupelicates, in which case I'm happy that we pass the actual string we want to delete here.